### PR TITLE
Enhanced tonie cards

### PR DESCRIPTION
**Pull request by CTeipen**
Blue frame marks a last played tonie
image

Information modal contains details of the tonie (hiding parts where no information is visible)
image
image
image

The edit modal allows editing the model and the source. You can write directly into the source intput field
image

On click of the directory icon next to the source input field, one can select a taf file
image

Only when source or model is changed, the save button is enabled and ready to save the changes
image

image

### CODE REVIEW

**Comment by SciLor** [comment]
Great small change =). Thx

File: src/components/tonies/TonieCard.tsx [hunk 1]
```diff
@@ -1,300 +1,469 @@
-import React, { useState, useEffect } from 'react';
-import { useTranslation } from 'react-i18next';
-import { Card, Button, Input, Popover, message, Modal } from 'antd';
-import { InfoCircleOutlined, PlayCircleOutlined, CloudSyncOutlined, RetweetOutlined, DownloadOutlined, EditOutlined, SaveOutlined, CloseOutlined, StarFilled } from '@ant-design/icons';
+import {
```
Comment (SciLor) [inline src/components/tonies/TonieCard.tsx:1]: There seem to be many changed lines due to a different auto format setting (There is nothing enforced currently).
What are you using?

**Comment by henryk86** [comment]
The success message set audio source. The last character is missing? ... .mp! Instead of expected ... .mp3!

Or is this just a "faked" image for demonstration purposes?

**Review by henryk86 — APPROVED** [review]

**Comment by SciLor** [comment]
Could you add a screenshot of the latest version?

**Comment by CTeipen** [comment]
> Could you add a screenshot of the latest version?

The screenshots above are the latest ones

**Comment by SciLor** [comment]
The formatting is still wrong (2 spaces, instead of 4 / line splits for several things) (See TonieCard.tsx)

**Comment by CTeipen** [comment]
> The formatting is still wrong (2 spaces, instead of 4 / line splits for several things) (See TonieCard.tsx)

honestly i'm confused about the single quotes. Is this correct this way?

**Comment by SciLor** [comment]
> > The formatting is still wrong (2 spaces, instead of 4 / line splits for several things) (See TonieCard.tsx)
> 
> honestly i'm confused about the single quotes. Is this correct this way?

I would stick to the style used in the file.

I mostly see double quotes for normal text string, single quotes for the imports and the special "`" for the strings with variables

**Comment by CTeipen** [comment]
> > > The formatting is still wrong (2 spaces, instead of 4 / line splits for several things) (See TonieCard.tsx)
> > 
> > 
> > honestly i'm confused about the single quotes. Is this correct this way?
> 
> I would stick to the style used in the file.
> 
> I mostly see double quotes for normal text string, single quotes for the imports and the special "`" for the strings with variables

I'm not sure if the formatter delivers a functionality for this.

**Comment by CTeipen** [comment]
I would want to use this state now. What is your opinion @SciLor ?

**Review by SciLor — APPROVED** [review]

**Comment by SciLor** [comment]
Post "noise" is now removed, thank you. Should I merge this state?

**Pull request merged**